doc/server/tutorial.rst

```diff
@@ -221,8 +221,13 @@ Aggregate sensors
 To provide a sensor which has its reading derived from that of a set of other
 sensors, such as a total, average or general "device status" sensor, subclass
 :class:`.AggregateSensor`, and implement
-:func:`.~AggregateSensor.update_aggregate` to compute the sensor reading from
+:func:`.!update_aggregate` to compute the sensor reading from
```

Comment:
I'm guessing you changed this because it wasn't making a link, but now you've got a literal `!` in the document. I think you possibly wanted this:
```suggestion
:meth:`~.AggregateSensor.update_aggregate` to compute the sensor reading from
```

Reply:
I don't see any `!`:
![image](https://user-images.githubusercontent.com/10864691/199459233-7ab95e93-ebd6-42d8-b967-24226ed8e8f2.png)
But still, having working links is nice.

Reply:
Odd, I definitely saw a `!` in my output. Possibly a different sphinx version.